File: macroslib/tests/expectations/document_generated_code.rs
foreigner_class!(
/// This is class Foo
class Foo {
    self_type Foo;
    /// Some documentation comment
    constructor Foo::new(_: i32, _: &str) -> Rc<RefCell<Foo>>;
    /// 1 Some documentation comment
    /// 2 Some documentation comment
    method Foo::f(&self, _: i32, _: i32) -> i32;
});

File: macroslib/tests/expectations/document_generated_code.cpp
r#"//This is class Foo
template<bool OWN_DATA>
class FooWrapper {"#;

r#"//Some documentation comment
    FooWrapper(int32_t a0, std::string_view a1) noexcept"#;

r#"//1 Some documentation comment
    //2 Some documentation comment
    int32_t f(int32_t a0, int32_t a1) const noexcept;"#;
